packages/playground/website/src/components/playground-configuration-group/index.tsx

```diff
@@ -157,7 +157,7 @@ export default function PlaygroundConfigurationGroup({
 
 	async function handleSubmit(config: PlaygroundConfiguration) {
 		const playground = await playgroundRef.current!.promise;
-		if (config.resetSite && config.storage === 'opfs-browser') {
+		if (config.resetSite && config.storage === 'opfs-browser' || config.storage === 'browser') {
```

Comment:
I'd add parentheses here:

```suggestion
		if (config.resetSite && (config.storage === 'opfs-browser' || config.storage === 'browser')) {
```

For the computer it's the same, but for a person it's less ambiguous.